vendor/vendor.json

```diff
@@ -9,10 +9,13 @@
 			"revisionTime": "2017-04-07T07:44:04Z"
 		},
 		{
-			"checksumSHA1": "32qRX47gRmdBW4l4hCKGRZbuIJk=",
+			"checksumSHA1": "oUPCyOhO8e61S2KVMJzM8sDnFdA=",
+			"origin": "github.com/sapk-fork/gitea-go-sdk/gitea",
```

Comment:
Yes this temporary and will be added in a PR in sdk. All the input and output format (used by api) are in sdk and I have already done one PR to move a struct in gitea to sdk.